cmd/peer-rest-server.go

```diff
@@ -269,6 +270,30 @@ func (s *peerRESTServer) LoadGroupHandler(w http.ResponseWriter, r *http.Request
 	}
 }
 
+// LoadIAMRefreshDataHandler - loads IAM refresh data.
+func (s *peerRESTServer) LoadIAMRefreshDataHandler(w http.ResponseWriter, r *http.Request) {
+	if !s.IsValid(w, r) {
+		s.writeErrorResponse(w, errors.New("Invalid request"))
+		return
+	}
+
+	objAPI := newObjectLayerFn()
+	if objAPI == nil {
+		s.writeErrorResponse(w, errServerNotInitialized)
+		return
+	}
+
+	cache := IAMCache{}
```

Comment:
To avoid setting nil pointer to some of the maps used in `cache` due to a notification to refresh, say when there are no groups or users yet.
```suggestion
	cache := newIAMCache()
```